tasks/settings.yml

```diff
@@ -17,6 +17,20 @@
     line: 'JENKINS_HOME={{ jenkins_home }}'
   register: jenkins_home_config
 
+- name: Set the user that Jenkins runs as
+  lineinfile:
+    dest: "{{ jenkins_init_file }}"
+    regexp: '^JENKINS_USER=.*'
+    line: 'JENKINS_USER={{ jenkins_process_user }}'
+  register: jenkins_process_user_config
+
+- name: Set the group that Jenkins runs as
+  lineinfile:
+    dest: "{{ jenkins_init_file }}"
+    regexp: '^JENKINS_GROUP=.*'
+    line: 'JENKINS_GROUP={{ jenkins_process_group }}'
+  register: jenkins_process_group_config
```

Comment:
This and jenkins_process_user_config will need checking in the service restart step at line 78ish ?

Reply:
Thanks for finding this @sf-nh. https://github.com/geerlingguy/ansible-role-jenkins/pull/210/commits/d73047b8804a507ca23b0eea6012fd8b5ed8f5cf should address this.